stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/ImageLoader.kt

```diff
@@ -38,6 +42,29 @@ public object ImageLoader {
             }
     }
 
+    public suspend fun getBitmapUri(
+        context: Context,
+        url: String,
+        transformation: ImageTransformation = ImageTransformation.None
+    ): Uri? = getBitmap(context, url, transformation)?.getLocalBitmapUri(context)
+
+    private fun Bitmap.getLocalBitmapUri(context: Context): Uri? {
+        var bmpUri: Uri? = null
```

Comment:
No need to create this `bmpUri` variable. You can inline the return with the `try` block and return the last statement there.